zorg/buildbot/builders/annotated/rise-riscv-build.sh

```diff
@@ -112,3 +112,33 @@ cmake --build stage2
 
 build_step "llvm-project check-all"
 cmake --build stage2 --target check-all
+
+# TODO: Evaluate running the test suite immediately after stage1 as it
+# executes very quickly and could provide rapid "fail fast" feedback.
+if [ ! -d llvm-test-suite ]; then
+  build_step "Cloning llvm-test-suite repo"
+  git clone --progress https://github.com/llvm/llvm-test-suite.git
+fi
+
+build_step "Updating llvm-test-suite repo"
+git -C llvm-test-suite fetch origin
+git -C llvm-test-suite reset --hard origin/main
+
+build_step "llvm-test-suite cmake"
+export QEMU_LD_PREFIX="$(pwd)/../rvsysroot"
+export QEMU_CPU="$BB_QEMU_CPU"
+cmake -G Ninja \
+  --toolchain=$(pwd)/stage1-toolchain.cmake \
+  -DTEST_SUITE_LIT=$(pwd)/stage1/bin/llvm-lit \
```

Comment:
Just want to check, if we don't specify `-C cmake/caches/O3.cmake` etc., does it default to building the release configuration? I think it's better if we can leave this as implicit since it doesn't look like ClangBuilder.py specifies it either (but also runs it via lnt: https://github.com/llvm/llvm-zorg/blob/3c627f0fa9da5566c88600b45e8293821a9fde2e/zorg/buildbot/builders/ClangBuilder.py#L560-L579)

Reply:
It does default to CMAKE_BUILD_TYPE=Release which defaults to `-O3 -DNDEBUG`.

I think it's probably good to avoid people having to check the default behaviour when reading the build script, so I'll go ahead and add an explicit `-DCMAKE_BUILD_TYPE=Release`.